.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would you please add `.swiftpm` to the repo root directory's `.gitignore` and remove this file from the PR?